modules/storages/app/services/projects/copy/storage_project_folders_dependent_service.rb

```diff
@@ -46,6 +46,12 @@ def source_count
     def copy_dependency(*)
       return unless state.copied_project_storages
 
+      GoodJob::Batches.enqueue(on_success: NotifyCopyCompletedJob, project_storages: state.copied_project_storages) do
+        state.copied_project_storages.each do |project_storages|
+          ::Storages::CopyProjectFoldersJob.enqueue(project_storages[:source], project_storages[:target])
+        end
+      end
```

Comment:
🔴 Did we talk about GoodJob not being part of 13.4 today in the daily? If yes, we cannot use it to enqueue batches. Am I missing something from the communication?

Reply:
Oh, this is enqueue as many jobs as there are project storages. Batches would give us a way to check for completion of all jobs and also re-organize the main `CopyProjectJob` around it.

Reply:
So there is no problem with using `GoodJob::Batches.enqueue` before our delayed jobs are refactored to GoodJob?

Reply:
In theory not, enqueue is part of `ActiveJob`s API, but this file just got deleted as it isn't needed anymore. XD